pkg/loop/internal/pb/relayer.proto

```diff
@@ -71,6 +72,11 @@ message PluginArgs {
   bytes pluginConfig = 2;
 }
 
+// NewEVMChain has reply parameters for [github.com/smartcontractkit/chainlink-common/pkg/loop.Relayer.NewEVMChain].
+message NewEVMChainReply {
+  uint32 EVMChainID = 1;
```

Comment:
We shouldn't be overloading this name

Reply:
Well if we call this an `EVMRelayer` instead of a `Chain`, then the problem goes away.

Reply:
But I think avoiding the extra indirection and client/server pair should be considered too. We don't really need a new thing - we just want to use a specialized subset of services